### PR TITLE
Log shipping/fargate

### PR DESCRIPTION
**Pull request by imnotashrimp**
# What changed

Drafted fargate shipping instructions. Definitely needs review, and there are some TODOs left in the doc.

----

## Pages to review

- https://deploy-preview-407--logz-docs.netlify.com/shipping/log-sources/fargate.html

## Remaining work

- [x] Technical review
- [x] Copy Review

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information: Marder
- [ ] New log shipping page - Fargate

### CODE REVIEW

File: _source/logzio_collections/_log-sources/fargate.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+Choose **Fargate**,
+and click **Next step** to continue.
+
+Scroll to the _Volumes_ section
+(near the bottom of the page)
+and click
+**Configure via JSON**.
+
+<div class="fpo fpo-2"></div>
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/fargate.md]: they're out now

File: _source/logzio_collections/_log-sources/fargate.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+- For **Subnets**, choose one subnet from your given options. TODO - we need more guidance here, I think.
+
+Click **Run task**.
+
+The logs created by the Fluent Bit shipper are in Cloudwatch
+under the `/aws/ecs/logzio-fargate-logs` log group.
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana).
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/fargate.md]: done

File: _source/logzio_collections/_log-sources/fargate.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+and click **Next step** to continue.
+
+Scroll to the _Volumes_ section
+(near the bottom of the page)
+and click
+**Configure via JSON**.
+
+<div class="fpo fpo-2"></div>
+
+Replace the default JSON
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/fargate.md]: done

File: _source/logzio_collections/_log-sources/fargate.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+  - container
+---
+
+This integration will send your AWS Fargate logs to FireLens.
+From there, Fluent Bit ships to Logz.io.
+
+#### Configuration
+
+**Before you begin, you'll need**:
+A cluster in a VPC with [Container Insights](https://docs.aws.amazon.com/AmazonECS/latest/userguide/cloudwatch-container-insights.html) enabled
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/fargate.md]: out now

File: _source/logzio_collections/_log-sources/fargate.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+data-source: Fargate
+contributors:
+  - imnotashrimp
+  - ronish31
+shipping-tags:
+  - aws
+  - container
+---
+
+This integration will send your AWS Fargate logs to FireLens.
```
[the remark this answers is not in the document]
Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/fargate.md]: done

**Comment by ronish31** [comment]
@imnotashrimp remove "taskRoleArn" from parameters.

File: _source/logzio_collections/_log-sources/fargate.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+| logConfiguration.options.Host | Your region's listener host. For more information on finding your account's region, see [Account region]({{site.baseurl}}/user-guide/accounts/account-region.html). |
+| logConfiguration.options.URI | {% include log-shipping/replace-vars.html token=true %} |
+{:.paramlist}
+
+###### Remaining parameters
+
+| Parameter | Description |
+|---|---|
+| executionRoleArn | Replace `<<AWS-ACCOUNT-ID>>` with your [AWS account Id](https://console.aws.amazon.com/billing/home?#/account). |
+| taskRoleArn | Replace `<<AWS-ACCOUNT-ID>>` with your [AWS account Id](https://console.aws.amazon.com/billing/home?#/account). |
```
Comment (imnotashrimp) [inline _source/logzio_collections/_log-sources/fargate.md]:
```suggestion
```

**Comment by imnotashrimp** [comment]
> @imnotashrimp remove "taskRoleArn" from parameters.

Done.

**Comment by idohalevi** [comment]
@imnotashrimp can we merge this?

**Comment by imnotashrimp** [comment]
> @imnotashrimp can we merge this?

Indeed

**Pull request merged**